demo/vagrant/Vagrantfile

```diff
@@ -69,13 +69,18 @@ nomad -autocomplete-install
 SCRIPT
 
 Vagrant.configure(2) do |config|
-  config.vm.box = "bento/ubuntu-18.04" # 18.04 LTS
+  config.vm.box = "generic/ubuntu2204" # 22.04 LTS, Jammy
```

Comment:
The unfortunately-named [generic](https://app.vagrantup.com/generic) org is not some "official" repo of common boxes, but one maintained by a company Lavabit LLC. We've been quite happy with `bento`, so we should keep it even if we decide to bump to the newer LTS.

Reply:
Ah yes, I did seem to miss that detail. I assumed generic was indeed some kind of generic provider and not ones maintained by another company. Looking into the Bento project, I also noticed that Bento is maintained by Chef which is owned by Progress Software.

For making the libvirt provider work, there are a few options I can think of:
1. Use something like [vagrant-mutate](https://github.com/sciurus/vagrant-mutate) to convert the image type, a bit of a hack
2. Wait for / help the Bento project with getting libvirt builds set up, some [context here](https://github.com/chef/bento/issues/1405). It seems like libvirt support is blocked by some infra issues on their end.
3. Use Ubuntu's official [cloud images](http://cloud-images.ubuntu.com/daily/server/jammy/current/) for Vagrant, although these unfortunately support only the [Virtualbox provider](https://app.vagrantup.com/ubuntu/boxes/jammy64)

Given all these solutions aren't ideal, I'll revert to the Bento version of 22.04 LTS.

Reply:
> Looking into the Bento project, I noticed that Bento is maintained by Chef which is owned by Progress Software.

Yes, it's also third party but we have a history of success with them (and use them in our own development environments, so they're well-tested).

Reply:
I'm fine to merge as-is and when upstream gets their libvirt situation sorted we'll get that for free.